admin/test/config/browserstack.yml

```diff
@@ -0,0 +1,75 @@
+server: "hub-cloud.browserstack.com"
+user: mohsen20
+authkey: ZyZxbzNCWSdnp1sgLCEz
```

Comment:
Keys have now been reset, turns out there is this - 
curl -u "<USERNAME>:<ACCESS_KEY>" -X PUT -d "{}" https://www.browserstack.com/automate/recycle_key.json